src/components/ecl-button/ecl-button.stories.ts

```diff
@@ -60,20 +60,28 @@ const getArgTypes = () => {
         'flip-vertical',
       ],
     },
+    showIndicator: {
```

Comment:
you're adding this here but without a default value it shows "set boolean". To be honest this indicator thingy is one of those that i haven't understood in ECL together with many others in the last months, it seems crazy that it only works if there is an icon and if the label is hidden, doesn't make any sense and it doesn't make much sense as a pure Ui element either, since there is no functionality associated with it. Anyway, you saw it in ECL and you added it here, we'll live with it.. :)